immersion/kubeflow_pipelines/walkthrough/solutions/lab-01_vertex.ipynb

```diff
@@ -0,0 +1,873 @@
+{
```

Comment:
<p>strictly speaking the grep should be <code>grep ^{ARTIFACT_STORE}$</code> or else it would match a partial string too. Also could we add a line above mentioning that this command creates the bucket if it does not exist?</p><p> </p><p>Furthermore, it could be the case that the bucket already exists but in another region. I think this could just be mentioned as text maybe, that you should make sure it's the same region -- I mean no need to write complicated code logic for this I think.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/mlops-on-gcp/pull/115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Done! (Added only the ^ and not the $ to the regex: the $ makes it fail because of the trailing '/')&gt; </p>

Reply:
<p>how about <code>grep ^{ARTIFACT_STORE}/$</code>  then?</p>

Reply:
<p>done!</p>